http/oidc/src/test/java/org/wildfly/security/http/oidc/OidcTest.java

```diff
@@ -55,6 +58,7 @@ public static void startTestContainers() throws Exception {
         assumeTrue("Docker isn't available, OIDC tests will be skipped", isDockerAvailable());
         KEYCLOAK_CONTAINER = new KeycloakContainer();
         KEYCLOAK_CONTAINER.start();
+        Runtime.getRuntime().addShutdownHook(new Thread(KEYCLOAK_CONTAINER::stop)); //remove before push
```

Comment:
I added this because I am using Podman to run a container. Should be deleted before the final push.

Reply:
This should be removed.